src/CLI/actioner/sonic-cli-show-techsupport.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+import sys
+import time
+import json
+import ast
+import re
+from scripts.render_cli import show_cli_output
+from rpipe_utils import pipestr
+import cli_client as cc
+import urllib3
+
+api = cc.ApiClient()
+urllib3.disable_warnings()
+plugins = dict()
+
+
+def register(func):
+    """Register sdk client method as a plug-in"""
+    plugins[func.__name__] = func
+    return func
+
+def call_method(name, args):
```

Comment:
Not used.

Reply:
I have deleted these lines, along with other unused lines.